### ISSUE TITLE
pbench-fio results.html unreadable at large scale

### ISSUE THREAD

**Issue by bengland2** [distributed-system-analysis/pbench#752]
with Tim Wilkinson's 1024-guest pbench-fio run, results.html becomes unreadable.  Need a different way to present this data.

**Comment by ndokos** [comment]
What does "unreadable" mean? I pinged Tim and he pointed me to the "data down below but not at top level" case where result.html is empty. Is this a different problem?

**Comment by bengland2** [comment]
yes exactly. fio is retrieving the logs, unlike before, but pbench-fio
throws errors while processing them.

**Comment by atheurer** [comment]
Unless we start adding some new html extensions, I don't think dynamically hiding the per-client results is all that feasible.  I suggest we create another result.html, like result-terse.html, which skips the per-client data in the table.  Users who have a large number of clients will just need to load the result-terse.html instead of result.html.

**Comment by bengland2** [comment]
The biggest problem with the result.html format was the horizontal scrolling required to view the results.  If we format as a vertical table then this problem can pretty much go away.  We can have the aggregate result at the top and the per-host results in a table that follows with 1 row per host.  It will be a big table but will be much more readable that way.  Agreed?

**Comment by ndokos** [comment]
This example contains just one iteration:  Transposing the table would change so it would have a dozen columns and a thousand rows, so you'd have to scroll through the thousand rows to get to the second iteration. It might be better than what we have now, but it's certainly no panacea. We have to make it much easier to navigate through the results: the way to that I believe is through the dashboard that @gurbirkalsi is working on. I'd like to see what this result looks like in the dashboard.

**Comment by bengland2** [comment]
True.  But the next step would be to compute some statistics on the  iops and latencies for each fio job, so you can see at a glance whether the system was fair about distributing resources to different pods or not.  I'd like to see @gurbirkalsi dashboard, is there a video recording of the demo that he did and does it address this?

**Comment by ashishkamra** [comment]
@bengland2 Gurbir will demo the enhancements to the dashboard in the Tools meeting on 5/29.
